src/main/java/org/fundacionjala/coding/danielM/bank_ocr/BankOcr.java

```diff
@@ -0,0 +1,170 @@
+package org.fundacionjala.coding.danielM.bank_ocr;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by Administrator on 3/10/2017.
+ */
+public class BankOcr {
+
+    /**
+     * This enum type will contain the key values.
+     */
+    private enum KEY {
+        ZERO(0), ONE(1), TWO(2), THREE(3),
+        FOUR(4), FIVE(5), SIX(6),
+        SEVEN(7), EIGHT(8), NINE(9);
+
+        /**
+         * Construct for Enum type.
+         *
+         * @param i of int type.
+         */
+        KEY(final int i) {
+        }
+    }
+
+    private Map<Integer, String> stringMap = new HashMap<>();
+    private static final int MODULUS_FACTOR = 11;
+    private static final int CORRECT_ACCOUNT_LENGTH = 9;
+    private static final int MULTIPLY_FACTOR = 9;
+
+    /**
+     * Constructor private.
+     */
+    public BankOcr() {
+        stringMap.put(KEY.ZERO.ordinal(),
```

Comment:
one observation
Is this constructor necessary? I don't think so.
will change the map? no, because only will map 10 number 0-9.

you can define the static values.
for instance
```java
private static final Map<Integer, String> STRING_MAP = new HashMap<>();
static {
    STRING_MAP.put(KEY.ONE.ordinal(),
                            "  |"
                         + "  |"
                         + "  |");
    STRING_MAP.put(KEY.TWO.ordinal(),
                            " _ "
                         + " _|"
                         + "|_ ");
}
```